### PR TITLE
[#97] Use the SRT protocol in samples

### PR DESCRIPTION
**Pull request by pobudzey**
* The Windows ffmpeg build is compiled with SRT enabled out of the box (option "--enable-libsrt") so everything should work fine
* We might need to use the srt-live-transmit tool in the future (as suggested in multiple tutorials I found online)

### CODE REVIEW

File: sender.py [hunk 1]
```diff
@@ -19,9 +19,9 @@ def send(file, ip):
             click.echo(f'Encoder with serial number {SENDER_SERIAL_NUMBER} has been successfully registered.')
     
     # This simply runs the following command
-    # ffmpeg -i <input> -f mpegts -v warning -stats udp://<ip>:23000
-    stream = ffmpeg.input(file)
-    stream = ffmpeg.output(stream, f'udp://{ip}:23000', f = 'mpegts', v = 'warning', stats = None)
+    # ffmpeg -re -i <input> -f mpegts -v warning -stats srt://<ip>:23000?pkt_size=1316
+    stream = ffmpeg.input(file, re = None)
+    stream = ffmpeg.output(stream, f'srt://{ip}:23000?pkt_size=1316', f = 'mpegts', v = 'warning', stats = None)
```
Comment (FGRCL) [inline sender.py:24]: We can modify the packet size? that's pretty cool. Is `1316` the default value?

File: sender.py [hunk 2]
```diff
@@ -19,9 +19,9 @@ def send(file, ip):
             click.echo(f'Encoder with serial number {SENDER_SERIAL_NUMBER} has been successfully registered.')
     
     # This simply runs the following command
-    # ffmpeg -i <input> -f mpegts -v warning -stats udp://<ip>:23000
-    stream = ffmpeg.input(file)
-    stream = ffmpeg.output(stream, f'udp://{ip}:23000', f = 'mpegts', v = 'warning', stats = None)
+    # ffmpeg -re -i <input> -f mpegts -v warning -stats srt://<ip>:23000?pkt_size=1316
+    stream = ffmpeg.input(file, re = None)
```
Comment (FGRCL) [inline sender.py:23]: In the ffmpeg options I see this options is to "Read input at native frame rate. " [link](https://ffmpeg.org/ffmpeg.html#toc-Advanced-options). I'm curious why we are passing None to it here?

**Review by FGRCL — APPROVED** [review]

File: sender.py (hunk 1 above)
Comment (pobudzey) [reply sender.py:24]: Yes we can! And it's not the default value but it's the value suggested by srt whenever you use ffmpeg and your input is a continuous data stream (file, camera, etc.) See --> https://github.com/Haivision/srt/blob/master/docs/srt-live-transmit.md#testing-considerations

File: sender.py (hunk 2 above)
Comment (pobudzey) [reply sender.py:23]: Good question - so with ffmpeg-python (the python wrapper around ffmpeg) whenever we want to pass an option that doesn't take an argument (such as -re, as opposed to -f for example) we have to include that option and set its value to None. It's a bit clunky but that's the way it works.

**Pull request merged**